e2e/run.js

```diff
@@ -50,6 +51,12 @@ puppeteer.launch({args}).then(async browser => {
     waitUntil: 'networkidle0'
   });
 
+  await page.on('console', msg => console.log('PAGE LOG:', msg.text()));
+
+  await page.addScriptTag({
```

Comment:
can we include this file via `<script>` tag on the `test-page.html` instead of injecting it?

Reply:
Well, when I added the file via a script tag, Puppeteer wasn't executing it. We either inject the file _or_ use `page.addScriptTag({ content: `string` });` I figured this approach was preferable because it kept logic appropriately separated.

Reply:
hm, have you figured out why puppeter wasn't executing it? Maybe it was 404 or it was executed too early / too late?

Reply:
As far as I recall, even a top-level `console.log` wasn’t firing. I’ll try again in a little bit and report back here.